src/pages/installing/uninstalling-ibm.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Uninstalling ACD"
+excerpt: "Uninstalling ACD."
+categories: installing
+slug: uninstalling
+toc: true
+---
+_Note: The following instructions, including the cloudctl command line utility, are for uninstalling IBM Annotator for Clinical Data Container Edition._
```

Comment:
The web console path should be in the uninstall ACD section and move only the cloudctl/cli uninstall to the IBM-specific page. You can also reference the web console uninstall path from the IBM uninstall page.